<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_registrar_movimiento</class>
 <widget class="QWidget" name="ventana_registrar_movimiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registrar movimientos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(52, 2, 2);

}

QFrame#frame_bottom{
    border-radius: 15px;
	background-image: url(:/images/images/fondoRojo.jpeg);
}

QFrame#frame_top{
	background-image: url(:/images/images/fondoRojo.jpeg);
	border-radius: 15px;
}

QLabel#label_titulo{
	color: white;
	background: transparent;
	font-size: 33px;
	font-family: Arial Black;
}

QLabel#label_error404{
	background: transparent;
}

QLabel{
	background: transparent;
	color: white;
	font-size: 20px;
	font: Ubuntu bold;
}

QLineEdit{
	color: black;
	background: white;
}

QPushButton{
color: white;
background: qlineargradient(spread:pad, x1:0.800995, y1:0.256182, x2:1, y2:0, stop:0 rgba(164, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
}

</string>
  </property>
  <widget class="QFrame" name="frame_top">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>621</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Sistema de Gesti√≥n Contable</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_bottom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>621</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_registrarMovimiento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>217</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar movimiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>217</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tipoMovimiento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de movimiento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_monto">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>80</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tipoMovimiento">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_monto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Monto                    $</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gobierno">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="source.qrc">:/images/images/parte_inferior1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_error404">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="source.qrc">:/images/images/registrarMovimiento.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
